packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -238,5 +251,24 @@ const getStyles = (theme: GrafanaTheme2) => {
       margin-left: -4px;
       color: ${theme.colors.text.secondary};
     `,
+    rightWrapper: css`
+      display: flex;
+      align-items: center;
+      margin-left: auto;
+    `,
+    shortcutIcon: css`
+      margin-right: ${theme.spacing(1)};
+    `,
+    withShortcut: css`
+      min-width: ${theme.spacing(10)};
+    `,
+    shortcut: css`
+      display: flex;
+      align-items: center;
+      gap: ${theme.spacing(1)};
+      margin-left: ${theme.spacing(1)};
```

Comment:
think this should be spacing 2, otherwise it looks a bit uneven with 16px spacing on the right side 

![image](https://user-images.githubusercontent.com/10999/217251358-fb61732a-bdef-43b0-8f03-b26a7370e7da.png)

Reply:
ah, you're right. will change